modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -116,6 +116,15 @@ Ciphers and the minimum TLS version of the configured security profile are refle
 
 * `WildcardsDisallowed`: Indicates only routes with a wildcard policy of `None` are admitted by the Ingress Controller. Updating `wildcardPolicy` from `WildcardsAllowed` to `WildcardsDisallowed` causes admitted routes with a wildcard policy of `Subdomain` to stop working. These routes must be recreated to a wildcard policy of `None` to be readmitted by the Ingress Controller. `WildcardsDisallowed` is the default setting.
 
+|`IngressControllerLogging`
+|* `logging` defines parameters for what should be logged where. If this field is empty, operational logs are enabled but access logs are disabled.
+* `access` describes how the client requests should be logged. If this field is empty, access logging is disabled.
+* `ContainerLoggingDestinationParameters` describes parameters for the Container logging destination type.
+* `destination` describes a destination for log messages.
+** `type` is type of destination for logs. It must be one of the following:
+*** `container` enables the Ingress Operator to configure the sidecar container named *logs* on the Ingress Controller Pod and configures the Ingress Controller to write logs to the sidecar. The logs are then available as container logs. The expectation is that the administrator configures a custom logging solution that reads logs from this sidecar. Using container logs means that logs may be dropped if the rate of logs exceeds the container runtime's or the custom logging solution's capacity.
```

Comment:
If it doesn't change the meaning, I would eliminate the contractions at the end of the sentence to be less of a distraction:
> Using container logs means that logs may be dropped if the rate of logs exceeds the container runtime _capacity_ or the custom logging solution capacity.